Math/Relation/_.kind2

```diff
@@ -0,0 +1,25 @@
+Math.Relation (t) : Type
+Math.Relation t = t -> t -> Type
+
+// Properties
+Math.Relation.Reflexive <t> (rel: Math.Relation t) : Type
+Math.Relation.Reflexive t rel = (a: t) -> rel a a
+
+Math.Relation.Transitive <t> (rel: Math.Relation t) : Type
+Math.Relation.Transitive t rel = (a: t) -> (b: t) -> (c: t) -> rel a b -> rel b c -> rel a c
+
+Math.Relation.Antisymmetric <t> (rel: Math.Relation t) : Type
+Math.Relation.Antisymmetric t rel = (a: t) -> (b: t) -> rel a b -> rel b a -> Prop.Equal a b
+
+// alternative definition of antisymmetry
+Math.Relation.Antisymmetric.alt <t> (rel: Math.Relation t) : Type
```

Comment:
hmm 🤔, i'm using it on Trait/Ord/make, it type checks correctly. is it because the checker can find it but it's not acessible on runtime?

Reply:
You are correct that it checks (and runs) correctly, but that is a bug.
If a function has already been loaded we don't check where it is coming from and this can lead to unexpected behaviour.
I'll be fixing this on the future, so please change this PR to avoid having to fix this file when I make that change to Kind.